plugins/cloudbuild/.eslintrc.js

```diff
@@ -1 +1,5 @@
-module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
+    rules: {
+      '@backstage/no-top-level-material-ui-4-imports': 'error',
+    },
+  });
```

Comment:
it's nice to configure your editor to automatically insert a newline at the end of files